src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  font-weight: 500;
+  margin-right: 20px;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+.nav__item:last-child {
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: rgba(6, 11, 53, 1);
```

Comment:
it's better to use hexadecimal numbers for color. it won't be confusing for other developers, because everyone uses hex as usual.